### PR TITLE
[1LP][RFR] Corrected is_displayed for Projects

### PR DESCRIPTION
**Pull request by vatsalparekh**
Purpose or Intent
=================
Corrected `is_displayed` and `wait_for`
{{ pytest: cfme/tests/configure/test_access_control.py --long-running -vvv -k tenant }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a364e65044d5ccff24e6b76f308ef9b4fef4b74e:

`cfme/configure/access_control.py`:
- :bangbang: Line 1231:101: **E501** *line too long (104 > 100 characters)*
- :bangbang: Line 1232:101: **E501** *line too long (106 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 794e9ff85284696a225e6fd3f631a669c8094e73:

`cfme/configure/access_control.py`:
- :bangbang: Line 1232:22: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d2276c591f65077171cd1a8c36f2ec20f2ad2310:

`cfme/configure/access_control.py`:
- :bangbang: Line 734:17: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/configure/access_control.py [hunk 1]
```diff
@@ -1228,7 +1229,12 @@ class TenantQuotaView(ConfigurationView):
     def is_displayed(self):
         return (
             self.form.template_cb.is_displayed and
-            self.title.text == 'Manage quotas for Tenant "{}"'.format(self.context['object'].name)
+            any((
```
Comment (mshriver) [inline cfme/configure/access_control.py]: **Required**

Use the contextual object to determine the correct title here (Tenant vs Project).  This is not a unique identification if we're on the correct page.

File: cfme/configure/access_control.py [hunk 2]
```diff
@@ -1294,7 +1301,8 @@ def is_displayed(self):
         return (
             self.form.accordions.accesscontrol.is_opened and
             any([self.form.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name),
-                self.form.title.text == 'Editing Project "{}"'.format(self.context['object'].name)])
+                 self.form.title.text == 'Editing Project "{}"'.format(
```
Comment (mshriver) [inline cfme/configure/access_control.py]: **Required**

I see `any()` was used here, before, but same as above I would like to see a unique identification of the view based on the contextual object.

**Comment by mshriver** [comment]
Added test selection for PRT, marking WIPTEST while its running

**Comment by rlbabyuk** [comment]
@vatsalparekh Please check the failures for 5.9.

**Comment by vatsalparekh** [comment]
@rlbabyuk it is failing due to https://bugzilla.redhat.com/show_bug.cgi?id=1511853

**Pull request merged**